index.html

```diff
@@ -35,7 +37,174 @@ <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
                 </ul>
             </div>
         </section>
-    </section>    
+    </section>
+
+    <!-- Main-section -->
+
+    <main class="main">
+
+        <!-- page-2 -->
+
+        <section class="page-2">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-1.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+            
+        <section class="page-3">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-2.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] As I already mentioned before, having only 1 `h1` is important for semantics and **SEO** reasons! Kindly change this heading 🤓